scripts/core/editor3/reducers/toolbar.jsx

```diff
@@ -143,13 +144,14 @@ const removeFormat = (state) => {
  * @ngdoc method
  * @name insertMedia
  * @param {Array} files List of media files to be inserted into document.
+ * @param {String} moveKey Block key
  * @description Inserts a list of media files into the document.
  */
-const insertMedia = (state, files = []) => {
-    var {editorState} = state;
+const insertMedia = (state, {files = [], moveKey = null}) => {
```

Comment:
Would be nicer if it were obvious from naming if the `moveKey` is the key of an item you are moving or the key of the target you are moving the item to.

Reply:
yep you're right, `targetBlockKey` would be better